src/test/resources/data/typicalDataBundle.json

```diff
@@ -173,6 +173,12 @@
       "name": "Unregistered Course",
       "timeZone": "UTC"
     },
+    "testingInstructorsDisplayedCourse": {
```

Comment:
Do you need to create a new course just for this?

Reply:
The reason I added another course specifically for this is that I need to test two workflows.

1) When a course has more than one instructor with `isDisplayedToStudents` set to true and editing one of them is a success. I used the course with id `idOfTypicalCourse1` 

2) When a course has only one instructor with `isDisplayedToStudents` set to true and editing that particular instructor throws an exception. Which is why i created this new course.